addons/website_slides/static/src/xml/website_slides_upload.xml

```diff
@@ -24,6 +24,9 @@
                 </a>
             </div>
         </div>
+        <a t-if="widget.moduleToInstall" id="wslides_import_survey_button" href="#" class="d-flex flex-column align-items-center mb-4 btn rounded border text-600 p-3">
+            Certificate your students by adding the feature
```

Comment:
Add the trophy icon + reword:
<i class="fa fa-trophy"></i> Evaluate and certificate your students. 
(on the sale line)